packages/foundation/src/atoms/core/foundations/index.css

```diff
@@ -0,0 +1,11 @@
+.o-ui-background {
```

Comment:
Shouldn't be any "o-ui" in foundation.

I don't think either there should be CSS classes in foundation, only vars

Reply:
these vars aka custom properties are set in themeProvider, I agree these should be atomic classes exposed by Tachyons down the line, I am removing this folder until I expose these classes as discussed.